Source/ACE/Entity/GenericObject.cs

```diff
@@ -0,0 +1,64 @@
+﻿// WeenieType.Generic
+
+using ACE.Entity.Actions;
+using ACE.Entity.Enum;
+using ACE.Network;
+using ACE.Network.GameEvent.Events;
+using ACE.Network.GameMessages.Messages;
+
+namespace ACE.Entity
+{
+    public class GenericObject : WorldObject
+    {
+        public GenericObject(AceObject aceObject)
+            : base(aceObject)
+        {
+        }
+
+        public GenericObject(ObjectGuid guid, AceObject aceObject)
+            : base(guid, aceObject)
+        {
+        }
+
+        ////public override void OnCollide(ObjectGuid playerId)
+        ////{
+        ////    // TODO: Implement
+        ////}
+
+        public override void OnUse(ObjectGuid playerId)
+        {
+            // TODO: Implement
+
+            if (CurrentLandblock != null)
+            {
+                ActionChain chain = new ActionChain();
+                CurrentLandblock.ChainOnObject(chain, playerId, (WorldObject wo) =>
+                {
+                    Player player = wo as Player;
+                    if (player == null)
+                    {
+                        return;
+                    }
+
+                    var errorMessage = new GameMessageSystemChat($"Generic OnUse reached, this object ({Name}) not programmed yet.", ChatMessageType.System);
+                    // var errorSound = new GameMessageSound(Guid, Sound.UI_GeneralError, 1);
+                    // var errorSound = new GameEventDisplayStatusMessage(player.Session, StatusMessageType1.Enum_058D);
+                    // var errorSound = new GameEventDisplayStatusMessage(player.Session, StatusMessageType1.Enum_0036);
+                    var sendUseDoneEvent = new GameEventUseDone(player.Session);
+
+                    player.Session.Network.EnqueueSend(errorMessage, sendUseDoneEvent);
+                });
+                chain.EnqueueChain();
+            }
+        }
+
+        public override void InternalOnUse(Session session)
+        {
```

Comment:
Could be bad naming, how about OnUse and OnUseContained or ContainedOnUse?

The difference being OnUse is called upon something in the landblock, the other is called when the player already has the object